Reply/app/src/main/res/color/star_statelist.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we name this file something like `color_email_star` or something that is a little closer to the existing color resource files?